### ISSUE TITLE
readme: tell that action factories require Http module to be added

### ISSUE THREAD

**Issue by fvgoto** [cerebral/cerebral-module-http#8]
A bit too naively, I supposed that [importing a http action factory](https://github.com/cerebral/cerebral-module-http/blame/master/README.md#L46) in some signal or chain was enough to use it (... as is the case e.g. for [`cerebral-addons/set`](https://github.com/cerebral/cerebral-addons/blame/master/README.md#L8).)
But it rather needs the `Http` to be [added as sub-(/dependent) module](https://github.com/cerebral/cerebral-module-http/blame/master/README.md#L20) beforehand.
Indeed, if we want some axios options, it is logical to add them at `Http` instantiation.

Maybe it would be helpful to the beginner to mention the module requirement explicitly on the [README](https://github.com/cerebral/cerebral-module-http/blame/master/README.md#L43) page.

And why not adding a check and a useful message in the [action factories](https://github.com/cerebral/cerebral-module-http/blob/master/get.js#L9)?
Something of this kind:

```
   var http = args.modules['cerebral-module-http'];
+  if( ! http) {
+    throw "Http action factories require 'cerebral-module-http' module to be added to controller or current module";
+  }
```

**Comment by christianalfoni** [comment]
Yeah, this is really good feedback! I completely agree :)

I am working on the new Cerebral update currently, but if you want to create a pull for this as well you are very welcome :)

**Comment by fvgoto** [comment]
Fixed by merged PR #11.
